bedrock/foundation/redirects.py

```diff
@@ -0,0 +1,11 @@
+from bedrock.redirects.util import redirect
+
+redirectpatterns = (
+
+    redirect(r'^foundation/trademarks/$', '/foundation/trademarks/policy/'),
```

Comment:
Is this redirect intended to forward `/foundation/trademarks/` to `/foundation/trademarks/policy`? Or am I reading it wrong?

Reply:
Yes, that was part of the requirement. Appeared to work as expected on demo. Incorrect?